micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/config/PropertyAbsentOrEnabledCondition.groovy

```diff
@@ -0,0 +1,31 @@
+package com.ofg.infrastructure.config
+
+import groovy.transform.CompileStatic
+import org.apache.commons.lang.BooleanUtils
+import org.springframework.context.annotation.Condition
+import org.springframework.context.annotation.ConditionContext
+import org.springframework.core.type.AnnotatedTypeMetadata
+
+@CompileStatic
+abstract class PropertyAbsentOrEnabledCondition implements Condition {
```

Comment:
It seems that there already is such a thing in Spring Boot - org.springframework.boot.autoconfigure.condition.ConditionalOnProperty. It's backed up by org.springframework.boot.autoconfigure.condition.OnPropertyCondition which is package scope so we can't use it.